packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/Convert.java

```diff
@@ -713,7 +713,7 @@ static String interpretPolylineOptions(
     sink.setEndCap(toCap(polyline.getEndCap(), assetManager, density));
     sink.setStartCap(toCap(polyline.getStartCap(), assetManager, density));
     sink.setGeodesic(polyline.getGeodesic());
-    sink.setJointType(polyline.getJointType().intValue());
+    sink.setJointType(polyline.getJointType().index);
```

Comment:
This isn't fixing the problem of implicitly relying on numeric values from two different sources lining up, it just changes exactly what the things that have to line up are; it would now silently break if the enum order were changed in the Pigeon file, or if Android changed the constants.

There should be a helper method that `switch`es over the enum and returns the SDK constants.